packages/ckeditor5-table/src/utils/ui/table-properties.ts

```diff
@@ -378,13 +379,15 @@ export function getLabeledColorInputCreator(
 		colorConfig: Array<NormalizedColorOption>;
 		columns: number;
 		defaultColorValue?: string;
+		colorPickerConfig: false | ColorPickerConfig;
 	}
 ) {
 	return ( labeledFieldView: LabeledFieldView, viewUid: string, statusUid: string ): ColorInputView => {
 		const colorInputView = new ColorInputView( labeledFieldView.locale!, {
 			colorDefinitions: colorConfigToColorGridDefinitions( options.colorConfig ),
 			columns: options.columns,
-			defaultColorValue: options.defaultColorValue
+			defaultColorValue: options.defaultColorValue,
+			colorPickerConfig: options.colorPickerConfig
```

Comment:
We need tests for `getLabeledColorInputCreator()` to check whether this config is accepted and actually passed down to the `ColorInputView`.